### PR TITLE
Improve public url_filter docs

### PR DESCRIPTION
**Pull request by will-moore**
This improves the docs for using webclient UI for public data. See discussion at https://www.openmicroscopy.org/community/viewtopic.php?f=5&t=7761&start=10#p15517

Here I've attempted to provide a url filter that blocks all the urls you might want to disable from a public facing site, while still allowing webclient UI.

Also I've removed the advice "You probably do not want the whole webclient UI to be publicly visible" since many people do want to do this, and we now support it better.

To test, please set the described url filter and see if webclient becomes "browse only" (can't create data or run scripts etc). Also see https://gist.github.com/will-moore/8315580 (do we want to bring this into our docs or link to it from anywhere)?

### CODE REVIEW

File: omero/developers/Web/PublicData.txt [hunk 1]
```diff
@@ -33,9 +33,7 @@ Here is how to go about setting this up on your OMERO.web install:
 
 -   Set a :property:`omero.web.public.url_filter` for which the OMERO.web
     public user is allowed to navigate.
-    Default: '^/(?!webadmin)' (Python reqular expression). You probably do not
-    want the whole webclient UI to be publicly visible (although you could do
-    this).
+    Default: '^/(?!webadmin)' (Python reqular expression).
```
Comment (mtbc) [inline omero/developers/Web/PublicData.txt]: regular

**Comment by aleksandra-tarkowska** [comment]
Could we say: Set a omero.web.public.url_filter (it uses Python reqular expression) for which the OMERO.web public user is allowed to navigate. There are three major use-cases:
then split the to 3 bullet points:

**Comment by aleksandra-tarkowska** [comment]
https://gist.github.com/will-moore/8315580 could go to https://github.com/openmicroscopy/openmicroscopy/tree/develop/examples/Web

**Comment by aleksandra-tarkowska** [comment]
otherwise looks good

**Comment by hflynn** [comment]
Can you get this fixed up please @will-moore ?

**Comment by will-moore** [comment]
@aleksandra-tarkowska Look OK?

**Comment by aleksandra-tarkowska** [comment]
Thank, looks good to me

**Comment by aleksandra-tarkowska** [comment]
@will-moore where are your examples? Will you do it in another PR?

**Comment by will-moore** [comment]
Yes, it has to be in another PR, since it is a different repository.

**Comment by aleksandra-tarkowska** [comment]
great, good to merge

**Comment by hflynn** [comment]
Is this 5.1 only, or does it need rebasing with the property links removing?

**Comment by will-moore** [comment]
@hflynn I think this would be useful to have for 5.0 too.

**Comment by hflynn** [comment]
Ok, can you rebase but with all the `:property:` links changed to ```` mark-up then please? Thanks.

**Pull request merged**